src/actions/files.js

```diff
@@ -16,6 +16,10 @@ export const FILE_NAMES = {
 const MAX_OPEN_REQUESTS = 15;
 const MAX_RETRIES = 5;
 
+// Connect uploads should be highest priority as they are user requested.
+// 0 is highest priority.
```

Comment:
```suggestion
// connect uploads should be high priority as they are user requested (lower is higher)
```
connect is the wordmark (lowercase) :slightly_smiling_face:

Reply:
you are right! fixed